src/__tests__/token-registry.e2e.test.ts

```diff
@@ -0,0 +1,295 @@
+import { ethers, Wallet } from "ethers";
+import { DeployTokenRegistryCommand } from "../commands/deploy/deploy.types";
+import { NetworkOption, WalletOrSignerOption } from "../commands/shared";
+import {
+  BaseTitleEscrowCommand,
+  TitleEscrowChangeHolderCommand,
+  TitleEscrowEndorseChangeOfOwnerCommand,
+  TitleEscrowNominateBeneficiaryCommand,
+} from "../commands/title-escrow/title-escrow-command.type";
+import { TokenRegistryIssueCommand } from "../commands/token-registry/token-registry-command.type";
+import { deployTokenRegistry } from "../implementations/deploy/token-registry";
+import { surrenderDocument } from "../implementations/title-escrow/surrenderDocument";
+import { issueToTokenRegistry } from "../implementations/token-registry/issue";
+import { ConnectedSigner, getWalletOrSigner } from "../implementations/utils/wallet";
+import { TransactionReceipt } from "@ethersproject/providers";
+import { acceptSurrendered } from "../implementations/title-escrow/acceptSurrendered";
+import { rejectSurrendered } from "../implementations/title-escrow/rejectSurrendered";
+import { changeHolderOfTitleEscrow } from "../implementations/title-escrow/changeHolder";
+
+import ganache from "ganache";
+import { endorseChangeOfOwner } from "../implementations/title-escrow/changeOwner";
+import { nominateBeneficiary } from "../implementations/title-escrow/nominateBeneficiary";
+import { endorseNominatedBeneficiary } from "../implementations/title-escrow/endorseNominatedBeneficiary";
+
+jest.mock("../implementations/utils/wallet", () => {
+  const originalModule = jest.requireActual("../implementations/utils/wallet");
+  return {
+    __esModule: true,
+    ...originalModule,
+    getWalletOrSigner: jest.fn(),
+  };
+});
+
+const accounts = {
+  mnemonic: "indicate swing place chair flight used hammer soon photo region volume shuffle",
```

Comment:
why is mnemonic, owner and receiver under `accounts`? would it be better to separate them ?